search-service/src/test/kotlin/com/egm/stellio/search/entity/web/EntityHandlerTests.kt

```diff
@@ -304,8 +310,19 @@ class EntityHandlerTests {
             )
     }
 
+    fun initializeRetrieveEntityMocks() {
+        val compactedEntity = slot<CompactedEntity>()
+
+        coEvery {
+            linkedEntityService.processLinkedEntities(capture(compactedEntity), any(), any())
+        } answers {
+            listOf(compactedEntity.captured).right()
+        }
```

Comment:
why is compactedEntity captured here in addition to mocking the behaviour of `processLinkedEntities()` ?